### PR TITLE
Added parameters

### PR DESCRIPTION
**Pull request by jcohenadad**

### CODE REVIEW

**Comment by jcohenadad** [comment]
@bpinsard i still need to add the spinal cord params-- i hope to find some time today to do it

**Comment by jcohenadad** [comment]
@alexfoias can you please help me i am struggling with time today  and they need that for todaty -- thanks

things to do: add spinal cord

File: source/MRI.md [hunk 1]
```diff
@@ -3,27 +3,28 @@
...
-  * T1-weighted MPRAGE sequence (duration 6:38 mn, TR=, TE=)
-  * T2-weighted sequence (duration 5:57 mn, ...)
-  * Diffusion-weighted sequence (duration 4:04 mn, ...)
-  * gradient-echo magnetization-transfer sequence (duration 3:34 mn, ...)
-  * gradient-echo proton density sequence (duration 3:34 mn, ...)
+  * T1-weighted MPRAGE 3D sagittal sequence (duration 6:38 min, TR = 2.4 s, TE = 2.2 ms, voxel size = 0.8 mm isotropic, R=2 acceleration)
+  * T2-weighted FSE (SPACE) 3D sagittal sequence (duration 5:57 min, TR = 3.2 s, TE = 563 ms, voxel size = 0.8 mm isotropic, R=2 acceleration)
+  * Diffusion-weighted 2D axial sequence (duration 4:04 min, TR = 2.3 s, TE = 82 ms, 57 slices, voxel size = 2 mm isotropic,  phase-encoding P-A, SMS=3 through-plane acceleration, b-max = 3000 s/mm^2). We've also added 5x b=0 scans with reversed phase encoding (A-P) for correcting susceptibility distortions.
```
Comment (jcohenadad) [inline source/MRI.md]: that's wrong-- need to be fixed @alexfoias

File: source/MRI.md [hunk 2]
```diff
@@ -3,27 +3,28 @@
...
 
 ### Brain anatomical sequences
 
  The parameters of the brain anatomical MRI sequences relevant for data analysis can be found in the [BIDS dataset](https://git.unf-montreal.ca/neuromod/), and included metadata. The acquisition parameters are identical for all anatomical sessions. The Siemens pdf exam card of the anatomical sessions can be found [here](./_static/mri/anatomical_protocol_2019-01-22.pdf), and is briefly recapitulated below. A standard (brain) anatomical session starts with a 21 s localizer scan, and then includes the following sequences:
-  * T1-weighted MPRAGE sequence (duration 6:38 mn, TR=, TE=)
-  * T2-weighted sequence (duration 5:57 mn, ...)
-  * Diffusion-weighted sequence (duration 4:04 mn, ...)
-  * gradient-echo magnetization-transfer sequence (duration 3:34 mn, ...)
-  * gradient-echo proton density sequence (duration 3:34 mn, ...)
+  * T1-weighted MPRAGE 3D sagittal sequence (duration 6:38 min, TR = 2.4 s, TE = 2.2 ms, voxel size = 0.8 mm isotropic, R=2 acceleration)
```
Comment (alexfoias) [inline source/MRI.md]: is it the gre_3d_T1w scan that can be found in the https://github.com/courtois-neuromod/cneuromod_docs/blob/master/source/_static/mri/anatomical_protocol_2019-01-22.pdf ?

Comment (jcohenadad) [reply source/MRI.md]: > @jcohenadad is it the gre_3d_T1w or the t1_mp2rage scan that can be found in the https://github.com/courtois-neuromod/cneuromod_docs/blob/master/source/_static/mri/anatomical_protocol_2019-01-22.pdf ?

it's the acq-0.8mm_T1w

but i've already entered and verified this, so it should be fine

File: source/MRI.md (hunk 1 above)
Comment (jcohenadad) [reply source/MRI.md]: to be more explicit: the only thing to change is: 
- remove 5x b=0
- add: same sequence with phase-encoding reversed

**Pull request merged**